experimental/saitypesextensions.h

```diff
@@ -48,6 +48,8 @@ typedef enum _sai_object_type_extensions_t
 
     SAI_OBJECT_TYPE_ENI,
 
+    SAI_OBJECT_TYPE_VIP_ENTRY,
```

Comment:
why adding in the middle and not at the end?

Reply:
It was autogenerated
Is it required to keep these enums in order?